### DISCUSSION TITLE
Zanata is now kinda deprecated... any alternatives for UI driven translations for muC?

### DISCUSSION THREAD

**Discussion by pskowronek** [mucommander/mucommander#1207] [General]
Zanata is now kinda deprecated... any alternatives for UI driven translations for muC?

```
**Maintenance-only Zanata Instance**

This service is in maintenance mode and new users and projects are not accepted, sorry.
```

**Comment by ahadas** [comment]
@pskowronek good catch, I didn't see that change in Zanata upstream. however, it would be interesting to see if anyone actually used it - I don't remember seeing activity there besides things I did there. I'm open to alternatives that would increase the amount of the relatively low contributions to the translation files.

**Comment by pskowronek** [comment]
@ahadas Recently I've stepped on this one: https://crowdin.com/pricing

**Reply by ahadas** [reply]
looks interesting, we can definitely check that
